update configuring sriov doc

zshi-redhat opened the pull request.

zshi-redhat commented:
/cc @jboxman @lamek @pliurh @zhaozhanqi

jboxman commented:
Hi @zshi-redhat, is this good? For what versions? Thanks!

zshi-redhat commented:
This is created against openshift-doc master for 4.4 release.
Once merged, it shall be cherry-picked to 4.3.

@pliurh PTAL.

pliurh commented:
LGTM

zhaozhanqi commented:
/lgtm

lamek commented on networking/multiple_networks/configuring-sr-iov.adoc:
```diff
@@ -9,39 +9,54 @@ toc::[]
 You can use the {product-title} console to install SR-IOV by deploying the
 SR-IOV Network Operator. The SR-IOV Network Operator creates and manages the
-components of the SR-IOV stack. The Operator provides following features:
+components of the SR-IOV stack. The Operator provisions following components:
```
Minor- the article "the is missing:
"The Operator provisions the following components:"

zshi-redhat replied:
Done

lamek commented on networking/multiple_networks/configuring-sr-iov.adoc:
```diff
@@ -9,39 +9,54 @@ toc::[]
 Here's the function of each above mentioned SR-IOV components.
 
+* The SR-IOV Operator is a Kubernetes Deployment that manages all SR-IOV
+components in cluster. It watches creation, update and deletion of Operator
```
Missing the article "a":
"...components in a cluster."

zshi-redhat replied:
Done

lamek commented on networking/multiple_networks/configuring-sr-iov.adoc:
```diff
@@ -9,39 +9,54 @@ toc::[]
+components in cluster. It watches creation, update and deletion of Operator
+Custom Resources and takes corresponding actions such as generating
+NetworkAttachmentDefinition Custom Resources for SR-IOV CNI, creating and
+updating configuration of SR-IOV network device plug-in etc.
```
Add a comma before "etc.":
"updating configuration of SR-IOV network device plug-in, etc."

zshi-redhat replied:
Done

zshi-redhat commented:
> @zshi-redhat Some very minor nits, and otherwise LGTM.
> 
> Can you please update and squash so we have a single commit?

Thanks @lamek for the review!
Updated and squashed into one commit.

lamek commented:
/cherrypick enterprise-4.3

lamek commented:
/cherrypick enterprise-4.4
